Integrations/integration-Wildfire.yml

```diff
@@ -180,13 +223,14 @@ script:
 
     function createUploadEntry(body, title, result) {
         var md = tableToMarkdown(title, body);
-        return {Type: entryTypes.note,
-                Contents: result,
-                ContentsFormat: formats.json,
-                HumanReadable: md,
-                ReadableContentsFormat: formats.markdown,
-                EntryContext: {
-                    "WildFire.Report(val.SHA256 == obj.SHA256)": prettifyUploadBody(body)
+        return {
+            Type: entryTypes.note,
+            Contents: result,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            ReadableContentsFormat: formats.markdown,
+            EntryContext: {
+                "WildFire.Report(val.SHA256 == obj.SHA256)": prettifyUploadBody(body)
```

Comment:
are we sure there is SHA256 and not just MD5 in this case also?

Reply:
this is the report entry, it's old code, and checking for md5 is indeed missing here.